### PR TITLE
[typo](docs) add additional explanation to the repeat function 

### PR DESCRIPTION
**Pull request by yagagagaga**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by zhangstar333 — DISMISSED** [review]

File: docs/zh-CN/docs/sql-manual/sql-functions/string-functions/repeat.md [hunk 1]
```diff
@@ -33,6 +33,13 @@ under the License.
 
 将字符串 str 重复 count 次输出，count 小于1时返回空串，str，count 任一为NULL时，返回 NULL
 
+:::tip
+repeat 函数默认最多输出 10000 个字符，可通过会话变量调整限制。
```
Comment (zhangstar333) [inline docs/zh-CN/docs/sql-manual/sql-functions/string-functions/repeat.md:37]: count = std::min<int>(count, repeat_max_num);

Comment (yagagagaga) [reply docs/zh-CN/docs/sql-manual/sql-functions/string-functions/repeat.md]: done

**Review by zhangstar333 — APPROVED** [review]

**Review by JNSimba — APPROVED** [review]
LGTM

**Comment by yagagagaga** [comment]
run buildall

**Pull request merged**